doc/game.md

```diff
@@ -5,20 +5,26 @@ The Game Loop module is a simple abstraction over the game loop concept. Basical
 * The render method takes the canvas ready for drawing the current state of the game.
 * The update method receives the delta time in seconds since last update and allows you to move to the next state.
 
-The class `Game` can be subclassed and will provide both these methods for you to implement. In return it will provide you with a `widget` property that returns the game widget, that can be rendered in your app.
+The class `Game` can be subclassed and will provide among others, these gameloop methods.
+At the same time, a `Game` subclass instance can be added to a Flutter widget tree via the `GameWidget`.
```

Comment:
`At the same time` is odd here, at the same time as what?

Reply:
Too many Linus tech tips videos. I'm talking just like him.